### ISSUE TITLE
Missing OpenSSL DLL On Windows 10

### ISSUE THREAD

**Issue by IJustWorkHere** [fluent/fluent-bit#3292]
## Bug Report

**Describe the bug**
After installing fluent-bit via the downloadable installer or executing it via the exe in the zip the application will fail to start with the error message
```
The code execution cannot proceed because libssl-1_1-x64.dll was not found.  Reinstalling the program may fix this problem.
```
To get things working I install OpenSSL from the link on wiki.openssl.org.  It would be great if this was included in the built artifact or if the docs listed this as a dependency somewhere.

**To Reproduce**
* Get a fresh Windows 10 install
* Perform the following
  ```
  > $url = 'https://fluentbit.io/releases/1.7/td-agent-bit-1.7.2-win64.zip'
  > Invoke-WebRequest $url -OutFile $HOME\td-agent-bit-1.7.2-win64.zip
  >  Get-FileHash $HOME\td-agent-bit-1.7.2-win64.zip
  > Expand-Archive -Path $HOME\td-agent-bit-1.7.2-win64.zip
  > Start-Process .\td-agent-bit-1.7.2-win64\td-agent-bit-1.7.2-win64\bin\fluent-bit.exe
  ```

**Expected behavior**
The fluent-bit program is run and dummy data is printed to stdout

**Screenshots**
![image](https://user-images.githubusercontent.com/16140964/112670453-25d7e100-8e1e-11eb-904c-4d1f401ab66e.png)

**Your Environment**

* Version used: 1.7.2
* Configuration:
* Environment name and version (e.g. Kubernetes? What version?):
* Server type and version: 
* Operating System and version: 
  * Windows 10 (64-bit w/x64 processor)
* Filters and plugins:

**Additional context**

**Comment by rifftual** [comment]
Duplicate of #3186

**Comment by fujimotos** [comment]
Thank you for the report. This is fixed in #3193. Please wait for
the next release...
